docs/source/monitoring_a_live_experiment.rst

```diff
@@ -106,7 +106,12 @@ extending the existing panes. This can be done customizing the
 your experiment class. Additionally, you can customize the display of the selected
 nodes customizing the :attr:`~dallinger.experiment.Experiment.node_visualization_html`
 method, or the :attr:`~dallinger.models.SharedMixin.visualization_html` property on your
-model class:
+model class.
+
+The dashboard database view can be customized by customizing the
+:attr:`~dallinger.models.SharedMixin.json_data` method on your model classes to add/modify
+data provided by each model to dashboard views, or by modifying the DataTables data returned
+by the :attr:`~dallinger.experiment.Experiment.table_data` method in your ``Experiment`` class.
```

Comment:
Maybe not! `table_data` has a half decent docstring, and since it produces a data structure for DataTables it's kind of hard to document beyond pointing to the Datatables options documentation. The `json_data` method of the `models.SharedMixin` definitely needs a decent docstring, but it's pretty trivial (just return some json serializable dict describing your object beyond what's included in the default json).

Reply:
I added a docstring to `json_data` (which is a pre-existing customization point) and it's now linked from there.